recipe/conda_build_config.yaml

```diff
@@ -16,9 +16,12 @@ blas_impl:
   - generic
 
 # https://github.com/conda-forge/.cirun
-github_actions_labels:          # [linux]
+github_actions_labels:          # [linux or win]
 - cirun-openstack-gpu-2xlarge   # [linux]
+- cirun-azure-windows-2xlarge   # [win]
 
 megabuild:
 - true      # [linux]
 - false     # [osx]
+- false     # [win]
```

Comment:
This means we're currently only building for a single python version. Rather than quintupling our builds (which would cause 20 jobs just for the windows server), I think we should fix the megabuild setup here.

Once there are passing builds for one python version, this shouldn't be that hard (I can help).